src/utils/theme.js

```diff
@@ -0,0 +1,19 @@
+function setTheme(themeName) {
+  localStorage.setItem('theme', themeName)
+  document.documentElement.className = themeName
+}
+
+function keepTheme() {
+  if(localStorage.getItem('theme')) {
+    let theme = localStorage.getItem('theme')
```

Comment:
let's use const :)

Reply:
> 
> 
> let's use const :)

Sure. Updating the changes now

Reply:
> 
> 
> let's use const :)

I have updated the changes. Please verify @faran4engg